.github/workflows/stale.yml

```diff
@@ -0,0 +1,34 @@
+# SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+# SPDX-License-Identifier: MIT
+name: Close stale issues
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+
+    if: ${{ github.repository_owner != 'nextcloud-gmbh' }}
+
+    permissions:
+      issues: write
+
+    steps:
+    - uses: actions/stale@v9
+      with:
+        repo-token: ${{ secrets.COMMAND_BOT_PAT }}
+        stale-issue-message: >
+          Hi there! It looks like this issue hasn’t seen much activity for a while and might need a bit more information to move forward.
+          If we don’t hear back soon, we may close it to keep things organized.
+          Thanks again for your contributions – we really appreciate it!
+        stale-issue-label: 'stale'
+        only-labels: 'needs info'
+        labels-to-remove-when-unstale: 'needs info,stale'
```

Comment:
Not sure if the *needs info* label should be rmoved automatically or by the maintainer who asked for information? When something else is commented, or another person comments, the information is not always provided and the label would have to be added back manually.

Reply:
Good catch, thank you!